src/lib/modules/job/job_test.go

```diff
@@ -0,0 +1,87 @@
+package job
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	config "github.com/bench-routes/bench-routes/src/lib/config_v2"
+	"github.com/bench-routes/bench-routes/tsdb/file"
+)
+
+var testapis []config.API = []config.API{
+	{
+		Name:   "API_1",
+		Every:  time.Second * 5,
+		Domain: "https://www.youtube.com",
+		Route:  "/watch?v=qqNenU-sQZI",
+		Method: "GET",
+		Headers: map[string]string{
+			"Content-Type": "application/json",
+		},
+	},
```

Comment:
Can we have another case like this, where the params are not in `route`, rather in the `params` field of `config.API`?